### DISCUSSION TITLE
How to further process exported XLSX file?

### DISCUSSION THREAD

**Discussion by andrej-gravik** [pxlrbt/filament-excel#172] [Q&A]
I use this plugin successfully to export excel data from filament.
Now I have requirement to save file to specific location (with database record) - id addition to downloading it.
Is there a way to work with generated export file, e.g. save it or send it via email?

**Comment by pxlrbt** [comment]
You could try to hook into the events of the underlying excel package. My package is not exposing any API or events for that.
